Bottom: c62a01a0bf616e40a4954ddb9e62ae9c44de5ef6
Top:    211afc2443ea9c1462f3e0b909def1f5a27af6be
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-01-03 11:00:04 +0800

Refresh of tuntap-xdp-transmission

---

diff --git a/include/linux/if_tun.h b/include/linux/if_tun.h
index 1cafdc2..08e6682 100644
--- a/include/linux/if_tun.h
+++ b/include/linux/if_tun.h
@@ -17,7 +17,7 @@
 
 #include <uapi/linux/if_tun.h>
 
-#define TUN_XDP_FLAG 0x1ULL
+#define TUN_XDP_FLAG 0x1UL
 
 #if defined(CONFIG_TUN) || defined(CONFIG_TUN_MODULE)
 struct socket *tun_get_socket(struct file *);
